docs/source/kiva/images.rst

```diff
@@ -0,0 +1,52 @@
+Kiva Image Rendering
+====================
+
+Drawing images in kiva is accomplished via
+:py:meth:`GraphicsContext.draw_image`. A unique feature of drawing images
+(relative to path drawing) is that you can apply an arbitrary translation and
+scaling to the image without involving the current transformation matrix.
```

Comment:
maybe add "via the `rect` parameter"? Just to be explicit

Also very well could be unnecessary so feel free to ignore.

Reply:
Done. I added a bit more detail about the `rect` parameter in a paragraph below.